src/main/webapp/app/exercises/shared/structured-grading-criterion/grading-instructions-details/grading-instructions-details.component.html

```diff
@@ -1,40 +1,108 @@
-<jhi-markdown-editor
-    #markdownEditor
-    style="height: 350px"
-    class="markdown-editor"
-    [markdown]="questionEditorText"
-    [showPreviewButton]="true"
-    [domainCommands]="domainCommands"
-    (markdownChange)="prepareForSave()"
-    (textWithDomainCommandsFound)="domainCommandsFound($event)"
->
-    <!-- Preview -->
-    <ng-container #preview>
-        <div class="pt-1 background-editor-high markdown-preview">
-            <div *ngFor="let criterion of exercise.gradingCriteria">
-                <label class="font-weight-bold" [innerHTML]="criterion!.title! | htmlForMarkdown"></label>
-                <table class="table table-bordered">
-                    <thead>
-                        <tr>
-                            <td class="font-weight-bold">Credits</td>
-                            <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.gradingScale">Grading Scale</td>
-                            <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.instrDescription">Instruction Description</td>
-                            <td class="font-weight-bold">Feedback</td>
-                            <td class="font-weight-bold" jhiTranslate="artemisApp.exercise.usageCount">Max Count in Score</td>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <tr *ngFor="let instruction of criterion.structuredGradingInstructions">
-                            <td style="width: 5%" [innerHTML]="instruction.credits"></td>
-                            <td style="width: 10%" [innerHTML]="instruction!.gradingScale! | htmlForMarkdown"></td>
-                            <td style="width: 40%" [innerHTML]="instruction!.instructionDescription! | htmlForMarkdown"></td>
-                            <td style="width: 35%" [innerHTML]="instruction!.feedback! | htmlForMarkdown"></td>
-                            <td style="width: 10%" [innerHTML]="instruction.usageCount"></td>
-                        </tr>
-                    </tbody>
-                </table>
+<div *ngIf="!exercise.gradingInstructionFeedbackUsed">
```

Comment:
Why does this look different to the view when editing the exercise before having grading instruction feedback used?
![image](https://user-images.githubusercontent.com/41366522/117764796-55f00d80-b22d-11eb-9341-bdf9d5ab930b.png) vs. 
![image](https://user-images.githubusercontent.com/41366522/117764850-67d1b080-b22d-11eb-8824-eff483091535.png)

I get why you need to let the user confirm changes after having already used the feedback, but beside, from a user perspective I would expect the edit interface to be the same.

Reply:
Also refresh button next to inputs are not aligned

Reply:
> Also refresh button next to inputs are not aligned 
resolved.

To Lorena>>
hmm, we decided to change view only if the instructions are used. Maybe the existing view may change in the future, for now we have decided to keep it. I took a note, I'll bring it up in the meeting... I expect some feedbacks about that after being used.